<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>357</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>522</width>
    <height>309</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Available actions:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="availableFilter">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clearButton1">
           <property name="text">
            <string/>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="availableActions">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QToolButton" name="moveUpButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Move action up</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QToolButton" name="removeButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Remove action</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QToolButton" name="addButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Add action</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QToolButton" name="moveDownButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Move action down</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Current actions:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="currentFilter">
           <property name="inputMask">
            <string/>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="clearButton2">
           <property name="text">
            <string/>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="currentActions">
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="setDefaultsButton">
         <property name="text">
          <string>Defaults</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Ok</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>&amp;Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>availableFilter</tabstop>
  <tabstop>clearButton1</tabstop>
  <tabstop>availableActions</tabstop>
  <tabstop>moveUpButton</tabstop>
  <tabstop>removeButton</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>moveDownButton</tabstop>
  <tabstop>currentFilter</tabstop>
  <tabstop>clearButton2</tabstop>
  <tabstop>currentActions</tabstop>
  <tabstop>setDefaultsButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>372</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton1</sender>
   <signal>clicked()</signal>
   <receiver>availableFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton2</sender>
   <signal>clicked()</signal>
   <receiver>currentFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
